src/core/init.js

```diff
@@ -73,12 +73,9 @@ var rootjQuery,
 
 					if ( elem ) {
 						// Inject the element directly into the jQuery object
-						this.length = 1;
 						this[0] = elem;
+						this.length = 1;
```

Comment:
Seemed strange to set the length before the element. I pulled several of the changes from @gibson042 's older PR because they just seemed so right. :smile: